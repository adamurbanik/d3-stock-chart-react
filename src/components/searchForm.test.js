import * as React from 'react';
import { shallow } from 'enzyme';
import SearchForm from './searchForm';

describe('<SearchForm />', () => {
  test('renders correctly', () => {
    const wrapper = shallow(<SearchForm />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.children.length).toBe(1);
  });
});
