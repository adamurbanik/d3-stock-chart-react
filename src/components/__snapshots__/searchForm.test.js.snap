// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchForm /> renders correctly 1`] = `
<Fragment>
  <div
    className="jsx-2694548524 container-form"
  >
    <form
      action="#"
      className="jsx-2694548524 search-form"
    >
      <input
        className="jsx-2694548524 search__input"
        name="search__input"
        onChange={[Function]}
        placeholder="Enter stock symbol i.e. AAPL, FB"
        type="text"
      />
      <button
        className="jsx-2694548524 search__button"
        onClick={[Function]}
      >
        <svg
          className="jsx-2694548524 search__icon"
        >
          <use
            className="jsx-2694548524"
            xlinkHref="app/img/sprite.svg#icon-magnifying-glass"
          />
        </svg>
      </button>
    </form>
    <JSXStyle
      dynamic={
        Array [
          "#f5f5f5",
          "#575c5b",
          "760px",
        ]
      }
      id="3423099469"
    >
      .container-form.__jsx-style-dynamic-selector{width:100%;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-pack:center;-webkit-justify-content:center;-ms-flex-pack:center;justify-content:center;}.search-form.__jsx-style-dynamic-selector{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-pack:center;-webkit-justify-content:center;-ms-flex-pack:center;justify-content:center;position:relative;-webkit-flex:0 0 100%;-ms-flex:0 0 100%;flex:0 0 100%;}.search__input.__jsx-style-dynamic-selector{width:100%;font-family:inherit;font-size:inherit;background-color:#f5f5f5;border:none;outline:none;padding:0.7rem 2rem;border-radius:100px;-webkit-transition:all 0.2s;transition:all 0.2s;}.search__input.__jsx-style-dynamic-selector:focus{background-color:#ededed;}.search__button.__jsx-style-dynamic-selector{position:absolute;right:15px;border:none;background-color:unset;-webkit-align-self:center;-ms-flex-item-align:center;align-self:center;outline:none;visibility:hidden;-webkit-transition:all 0.6s;transition:all 0.6s;opacity:0;}.show.__jsx-style-dynamic-selector{visibility:visible;opacity:1;}.search__icon.__jsx-style-dynamic-selector{height:1.3rem;width:1.3rem;fill:#575c5b;}@media screen and (min-width:760px){.search-form.__jsx-style-dynamic-selector{-webkit-flex:0 0 50%;-ms-flex:0 0 50%;flex:0 0 50%;}}
    </JSXStyle>
  </div>
</Fragment>
`;
